docs/user-guide/options.md

```diff
@@ -51,6 +51,27 @@ When a rule relies on the deprecated [`context`](../developer-guide/rules.md#con
 - a scoped disable comment, e.g. `/* stylelint-disable color-named */`, any problems reported by the scoped rule will not be automatically fixed anywhere in the source
 - an unscoped disable comment, i.e. `/* stylelint-disable */`, the entirety of the source will not be automatically fixed for that rule
 
+## `computeEditInfo`
+
+CLI flag: `--compute-edit-info, --cei`
+
+Compute edit information for autofixable rules.
+
+When enabled, warnings from autofixable rules will include a `fix` property containing:
+
+- `range` - the start and end indices of the text to replace
+- `text` - the replacement text
+
+The edit information will not be computed when:
+
+- the [`fix` option](#fix) is enabled
+- a rule's fix is disabled:
+  - in the configuration object, e.g. `"rule-name": [true, { disableFix: true }]`
+  - using configuration comments, e.g. `/* stylelint-disable rule-name */`
```

Comment:
```suggestion
  - using configuration comments, e.g. `/* stylelint-disable rule-name */` (if [`ignoreDisables`](#ignoredisables) is `false`)
```

Do we need a test for that?
i.e. `ignoreDisables: true` + configuration comment

Reply:
@jeddy3 is this too obvious?